tokens/test/erc-721/Rarible.test.js

```diff
@@ -179,6 +189,31 @@ contract("ERC721Rarible", accounts => {
     assert.equal(await token.isApprovedForAll(accounts[1], proxyLazy.address), true);
   });
 
+  it("check Royalties IERC2981", async () => {
+    testRoyaltyV2981Calculate = await TestRoyaltyV2981Calculate.new();
+
+    const minter = accounts[1];
+    let transferTo = accounts[2];
+    let royaltiesBeneficiary1 = accounts[3];
+    let royaltiesBeneficiary2 = accounts[4];
```

Comment:
we could probably check the case with only 1 beneficiary

Reply:
add tests:
1) 1 beneficiary
2) 0 beneficiary
3) 3 beneficiary with zero fees